messages/service/service.go

```diff
@@ -0,0 +1,92 @@
+package service
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+
+	"dont-slack-evil/apphome"
+
+	"github.com/slack-go/slack"
+	"github.com/slack-go/slack/slackevents"
+)
+
+var slackVerificationToken = os.Getenv("SLACK_VERIFICATION_TOKEN")
```

Comment:
I don't think services will have access to env variable. Does this work?

Reply:
Yes

Reply:
If you call the service through `messages/main.go` yes this layer will have access to env variables, because serverless will have injected the env variables (defined in `serverless.yml`)

But if you write a test for this service, and run it without a serverless command, I believe that you will find that the `slackVerificationToken` will be undefined.

Reply:
I don't plan on running unit tests through serverless commands.
I would prefer mocking the Slack calls and results.

E2E tests, on the contrary, can make use of serverless.

Reply:
Exactly, so my point is, don't put env variables in services, because they can only be read through serverless. If you run unit tests on this, it will fail

Reply:
They can also be read through the env directly (we can run the go tests with specific env variables)

Or what we can do is write a `config` file, which will make use of the env  variables. And the services will make use of this config, not env